src/main/java/org/folio/edge/domain/exception/CirculationProcessException.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.edge.domain.exception;
+
+public class CirculationProcessException extends RuntimeException {
```

Comment:
Why do we need this exception? Is it thrown anywhere?

Reply:
I suppose you are right